src/models/ihapijsStarterServerConfig.ts

```diff
@@ -8,6 +8,7 @@ export interface IHapijsStarterServerConfig {
     port: number;
     cors: boolean;
     corsOrigins: string[];
+    corsHeaders: string;
```

Comment:
Can the property be renamed to something like `accessControlAllowHeaders`? Looking at this outside of this project I'd have no idea what it means.